### PR TITLE
Add forbes.com

### LINKED ISSUE #583: New site request: Forbes.com
### New site requests - Before submitting an issue please read the following:

Only large or major sites will be considered. For small sites try the [Cookie Remover](https://chrome.google.com/webstore/detail/cookie-remover/kcgpggonjhmeaejebeoeomdlohicfhce) extension to remove cookies (might not work though). Also try installing uBlock Origin.

1. Visit an article on the site you want to bypass the paywall for and copy the article title.
2. Open up a new incognito window (Ctrl+Shift+N) and paste the article title into Google.
3. Click on the same article from the Google search results page. If it loads without a paywall you can submit a request here, otherwise my extension cannot bypass it either.

Usually premium articles cannot be bypassed as they are behind a hard paywall.

### PR DESCRIPTION
**Pull request by aaronzdavis**
Resolves #583

### CODE REVIEW

**Comment by kimkha** [comment]
Duplicates #562?

File: src/js/contentScript.js [hunk 1]
```diff
@@ -424,6 +424,17 @@ if (matchDomain('thewrap.com')) {
     }
 }
 
+if (matchDomain('forbes.com')) {
+    const paywall = document.querySelector('.fbs-auth__adblock');
```
Comment (kimkha) [inline src/js/contentScript.js:428]: Is this an adblock thingy?

**Comment by llacb47** [comment]
I get no paywall or anti-adblock on forbes.

(by the way, you don't need this extension for forbes, it works just fine with vanilla uBo)

**Comment by kimkha** [comment]
Interesting, I just checked with forbes.com, there is no paywall at all. This PR should be closed, I guess.

**Pull request closed without merging**